source/Lexer.cpp

```diff
@@ -656,6 +658,10 @@ TokenKind Lexer::lexDirective(Token::Info* info) {
     }
 
     info->extra = getDirectiveKind(lexeme().subString(1));
+    if (!onNewLine && std::get<SyntaxKind>(info->extra) != SyntaxKind::MacroUsage) {
+        // All directives other than a macro usage must be on their own line
+        addError(DiagCode::DirectiveNotFirstOnLine, currentOffset() - 1);
```

Comment:
Ah ok. I used this value because we used it in the stray grave case, but that one got to use the knowledge that currentOffset() - 1 = start. Will change.